### ISSUE TITLE
adjust learning rate

### ISSUE THREAD

**Issue by berna-ylmz** [IDKiro/action-recognition-pytorch#12]
Hello,
In adjust_learning_rate, Can you explain "**if not epoch % args.lr_step and epoch**" ?  
```
def adjust_learning_rate(optimizer, epoch):
	if not epoch % args.lr_step and epoch:
		for param_group in optimizer.param_groups:
			param_group['lr'] = param_group['lr'] * 0.1
	return optimizer
```

**Comment by IDKiro** [comment]
adjust lr for epoch `N *  args.lr_step`  but not for epoch `0`
`if not epoch % args.lr_step and epoch` == `if (not epoch % args.lr_step) and epoch`
0 is False and 1, 2, 3, 4, ... are all True.

**Comment by berna-ylmz** [comment]
Thank you.

**Comment by Berna-lab** [comment]
Finally, if I select 1e-4 for learning rate , it never enters the If loop.
